src/Reveal.Sdk.Dom.Tests/Data/DataSources/HostDataSourceFixture.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Reveal.Sdk.Dom.Data;
+using System;
+using Xunit;
+
+namespace Reveal.Sdk.Dom.Tests.Data.DataSources
+{
+    public class HostDataSourceFixture
+    {
+        [Fact]
+        public void GetHost_ReturnSameValue_WithSetValue()
+        {
+            // Arrange
+            var hostDataSource = new HostDataSource();
+            var expectedHost = "test-host";
+
+            // Act
+            hostDataSource.Host = expectedHost;
+
+            // Assert
+            Assert.Equal(expectedHost, hostDataSource.Host);

```

Comment:
Also pls validate that the `Properties` is correctly set.

Reply:
Added